linkerd/app/inbound/src/policy/config.rs

```diff
@@ -52,7 +53,10 @@ impl Config {
                     let c = control.build(dns, metrics, identity).new_service(());
                     Api::new(workload, c).into_watch(backoff)
                 };
-                Store::spawn_discover(default, ports, watch)
+                // TODO(eliza): should the idle timeout for policy discovery be
+                // configurable? One minute is totally arbitrary...
+                let idle_timeout = Duration::from_secs(60);
```

Comment:
@olix0r do you think this seems reasonable or should we change it?

Reply:
I think we should use the `cache_max_idle_age` from the top-level config

Reply:
sounds good, will do!